data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		not outfits "Microbot Grinder"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life signs aboard, most of which read as likely intelligent, with many thousands more minor life signs alongside them.`
+			`	Some of these minor life signs are flora, such as the tree-like things you saw in the massive dome atop the craft, as well as a wide variety of fauna. The rest, though, seem to be large orbs found throughout the craft, mostly in large groupings. A crew member identifies the orbs as eggs - undoubtedly Korath eggs.`
+			`	As you continue to scan this massive vessel, your computer's analysis makes it clear that a ship of this scale would require an actual army spending weeks of constant fighting to even have a chance of capturing it and an equally large workforce to fully pilot it. A thorough and time-intensive sweep would be absolutely essential to prevent surviving Korath from engaging in guerrilla campaigns against your crew or even just sabotaging vital components.`
```

Comment:
For the Thorough bit, you would only really need one sweep to eliminate the Korath, the point of that is to show that, simply taking out the initial defenders and getting the bridge like most ships, won't completely eliminate the threat since the sheer amount of additional space the ship has means plenty of places for stragglers to hide and sabotage from.
Most ships don't have entire living spaces aboard them.

Reply:
I like the first rewording, not sure about changing a single sweep to multiple, it feels like an unnecessary change, plus a single Thorough and time-intensive sweep should be good enough, once the left over Korath are gone the problem is solved for good which is what that line is meant to imply.

Reply:
Edited to remove that change